pudb/debugger.py

```diff
@@ -1109,6 +1110,17 @@ def examine_breakpoint(w, size, key):
                 else:
                     self.update_breakpoints()
 
+        def breakpoint_location(w, size, key):
```

Comment:
I think this is largely identical to `show_breakpoint`, bound to `Enter` in that context.

Reply:
Hmm, am I crazy? I was pressing Enter before, but it opened the dialog. I just tested it now and it does what I want. We definitely don't need this PR if Enter already does this.

Reply:
Oh this was changed recently and I was using the released version. Sorry for the noise. This can be closed.